requirements/common.txt

```diff
@@ -1,6 +1,6 @@
 asgi-redis==1.4.3
-boto3==1.9.88
-botocore==1.12.88
+boto3==1.13.16
```

Comment:
Just a caveat here, we need to make sure that this upgrade doesnt break any existing flow

Reply:
@vkartik97  support for EKS APIs have been recently added to boto3. So had to upgrade it.

Reply:
Sure. We just need to take care that it doesnt break anything 👍